src/os/bluestore/aio.cc

```diff
@@ -47,6 +48,8 @@ int aio_queue_t::submit_batch(aio_iter begin, aio_iter end,
     assert(r > 0);
     done += r;
     left -= r;
+    attempts = 16;
```

Comment:
suggest to use some symbolic constants..

Reply:
This kind of back-off mechanism became quite popular these times. We've gotten something very similar with the `OpHistory` thread. It might deserve a small but generic helper with `reset()` method to avoid repetitions like this one. Though, for the sake of backportability, I would prefer to have it separated from the fix.